5_references/50_generic_methods.txt

```diff
@@ -1,3 +1,7 @@
+ifdef::webhelp[]
+
 === Generic methods
 
-include::../generic_methods.asciidoc[]
\ No newline at end of file
+include::../generic_methods.asciidoc[]
+
+endif::webhelp[]
```

Comment:
Why can't we have the generic methods in the PDF too? It is build from asciidoc sources, what is wrong with this file?

I read the ticket, but there is no information in there. Please *always* explain the original problem in your tickets. Just having the solution is like only reading the second half of a book - it doesn't mean much to the reader.